inflation/main.py

```diff
@@ -37,24 +46,44 @@ def collect_db_stats(db_path):
     """
     db_stats = {}
     for entry in os.listdir(db_path):
-        if os.path.isfile(os.path.join(db_path, entry)) and entry.endswith(
-            ".json"
-        ):
+        entry = os.path.join(db_path, entry)
+        if os.path.isfile(entry) and entry.endswith(".jsonl"):
             with open(entry) as f:
-                db_stats[entry] = len(json.load(f))
+                db_stats[entry] = len(f.readlines())
```

Comment:
bunlarin farki nedir?

Reply:
Bu kritik. Unutma bizim dosyalar JSON değil, jsonl

Reply:
O yüzden direk her satır json.loads denmeli. Senin yaptığın gibi tek satırdan fazla olan dosyalar açılmaz. Bir JSON kaydetmiyoruz, her satır için bir JSON kaydediyoruz